src/ntpclient.cpp

```diff
@@ -7,21 +7,18 @@
 #include "timedata.h"
 #include "util.h"
 
-#include <sstream>
 #include <iomanip>
+#include <sstream>
 
 using namespace boost;
 using namespace boost::asio;
 
-bool CNtpClient::getTimestamp(uint64_t &timeRecv)
+bool CNtpClient::getTimestamp(uint64_t& timeRecv)
```

Comment:
Why is this line changed?

Reply:
I think a few things could have been messed with by a code formatter that was probably bundled with the C++ extension for VSCode I installed. I can revert these changes if you want